dojo/settings/settings.dist.py

```diff
@@ -1343,7 +1343,7 @@ def saml2_attrib_map_format(din):
     "Invicti Scan": ["title", "description", "severity"],
     "Checkmarx CxFlow SAST": ["vuln_id_from_tool", "file_path", "line"],
     "HackerOne Cases": ["title", "severity"],
-    "KrakenD Audit Scan": ["description", "mitigation", "severity"],
```

Comment:
This would create some churn for folks that didn't know about this change. Please revert this change

Reply:
have you considered the use case of someone updating dedupe fields via the `local_settings.py` file or environment variables? We have found that making these sorts of changes through a migration doesn't often go so smoothly. I would feel more comfortable not changing any dedupe settings and knowing user data will not be impacted. Especially since the mitigation field is not changing in this PR

A follow up question: do we want to update the mitigation field in all reimports? I would think no as we cannot make an accurate guess as to whether the user had changed the mitigation of a finding.